badge_server/main.py

```diff
@@ -339,9 +304,14 @@ def one_badge_image():
 
     status, _, _, _ = _get_all_results_from_cache(package_name)
     color = STATUS_COLOR_MAPPING[status]
-    url = URL_PREFIX + '{}-{}-{}.svg'.format(badge_name, status, color)
 
-    response = flask.make_response(requests.get(url).text)
+    response = flask.make_response(
+        pybadges.badge(
+            left_text=badge_name,
+            right_text=status,
+            right_color=color,
+            whole_link=url_prefix + flask.url_for('one_badge_target',
```

Comment:
Could you extract the whole_link to a separate line?

Reply:
Done